test/docs.jl

```diff
@@ -349,12 +384,7 @@ end
 
 @doc "This should document @m1... since its the result of expansion" @m2_11993
 @test (@doc @m1_11993) !== nothing
-let d = (@doc @m2_11993)
-    @test docstring_startswith(d, doc"""
-    No documentation found.
-
-    `@m2_11993` is a macro.""")
-end
+@test (@doc @m2_11993) === nothing
```

Comment:
Yes, it now acts the same as searching for method docs when the signature doesn't match, returning `nothing`. I think I might be able to get the message back though, I'll have a look tomorrow.